invenio_files_rest/helpers.py

```diff
@@ -166,10 +176,15 @@ def send_stream(stream, filename, size, mtime, mimetype=None, restricted=True,
     # Set cache-control
     if not restricted:
         rv.cache_control.public = True
+        # get the value of the flask config variable "SEND_FILE_MAX_AGE_DEFAULT" and use it to set how long (in
+        # seconds) a file can be cached before it is considered stale and the client should fetch the newest version.
```

Comment:
I would go a bit softer with the details, since these might change also in the future :)

```suggestion
        # See flask config variable "SEND_FILE_MAX_AGE_DEFAULT"
        # https://flask.palletsprojects.com/en/2.1.x/api/#flask.Flask.get_send_file_max_age
```

Reply:
In that case, I trust that Flask docs keep their links stable and explain in more detail (or point to other resources), vs. us maintaining details of how browser-side caching works. If this breaks in the future, of course we'll have to fix anyways, but I wouldn't also add into it maintaining a comment regarding how caching works, but delegate to [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age)

Reply:
that is fair 😄 , made the change.